Copilot/find instances of cite (#294)
symbiotic-engineering/MDOcean: pubs/renewable-energy-mdo/sections/methods.tex

\end{figure*}

\ifdefined\DISSERTATION

>     The module structure is shown in the XDSM diagram of \Cref{remdo:fig:n2}, a standard MDO representation \citet{lambe_extensions_2012} in which modules lie on the diagonal, inter-module data dependencies occupy the off-diagonal, and design-variable inputs and objective and constraint outputs enter and exit at the edges.

Author: ```suggestion
    The module structure is shown in the XDSM diagram of \Cref{remdo:fig:n2}, a standard MDO representation \citep{lambe_extensions_2012} in which modules lie on the diagonal, inter-module data dependencies occupy the off-diagonal, and design-variable inputs and objective and constraint outputs enter and exit at the edges.
```

\else
    \Cref{remdo:fig:n2} shows the module structure as an XDSM diagram \citep{lambe_extensions_2012}: design variables enter at the edges, data passes feed-forward between the modules on the diagonal, and the objective and constraint outputs exit at the left.
    The disciplines couple through shared design variables (most prominently the bulk dimensions, which feed geometry, hydrodynamics, and structures) and through feed-forward output dependence, with no feedback coupling, so a single pass evaluates each design apart from internal iteration in the dynamics module.
    Because this coupling is non-convex and non-monotonic, the modules cannot be optimized sequentially without sacrificing optimality, so all design variables are driven concurrently by a single optimizer, which is the central premise of the formulation in \Cref{remdo:sec:formulation}.
\fi

MDOcean adopts an implicit multiple-discipline-feasible (MDF) analysis architecture, in which an analytically-derived optimal control condition is solved implicitly inside the dynamics iteration rather than introducing control variables into the outer optimization.

> The reader is referred to \citet{mccabe_development_2026} for the full module-level assumptions, the control-architecture taxonomy and rationale, the constrained optimal control derivation, and the validation and benchmarking results that establish MDOcean's accuracy and \resultsRE[simRuntime] per-evaluation runtime (the property that makes the present optimization computationally feasible).

Author: ```suggestion
\citet{mccabe_development_2026} provides the full module-level assumptions, the control-architecture taxonomy and rationale, the constrained optimal control derivation, and the validation and benchmarking results that establish MDOcean's accuracy and \resultsRE[simRuntime] per-evaluation runtime (the property that makes the present optimization computationally feasible).
```
